src/masternodes/mn_rpc.cpp

```diff
@@ -3081,39 +3225,174 @@ UniValue isappliedcustomtx(const JSONRPCRequest& request) {
+                                                                              "    It means that we will select first accounts with maximal sum of neccessary token amounts."
+                    },
+                },
+                RPCResult{
+                       "\"hash\"                  (string) The hex-encoded hash of broadcasted transaction\n"
+                },
+                RPCExamples{
+                        HelpExampleCli("sendtokenstoaddress", "'{}' "
+                                                    "'{\"dstAddress1\":\"1.0@DFI\",\"dstAddress2\":[\"2.0@BTC\", \"3.0@ETH\"]}' \"crumbs\"") +
+                        HelpExampleCli("sendtokenstoaddress", "'{\"srcAddress1\":\"2.0@DFI\", \"srcAddress2\":[\"3.0@DFI\", \"2.0@ETH\"]}' "
+                                                    "'{\"dstAddress1\":[\"5.0@DFI\", \"2.0@ETH\"]}'")
+                        },
+    }.Check(request);
+
+    if (pwallet->chain().isInitialBlockDownload()) {
+        throw JSONRPCError(RPC_CLIENT_IN_INITIAL_DOWNLOAD, "Cannot create transactions while still in Initial Block Download");
+    }
+    pwallet->BlockUntilSyncedToCurrentChain();
+
+    RPCTypeCheck(request.params, {UniValue::VOBJ, UniValue::VOBJ, UniValue::VSTR}, false);
+
+    CAnyAccountsToAccountsMessage msg;
+
+    msg.to = DecodeRecipients(pwallet->chain(), request.params[1].get_obj());
+    const CBalances sumTransfersTo = SumAllTransfers(msg.to);
+    if (sumTransfersTo.balances.empty()) {
+        throw JSONRPCError(RPC_INVALID_PARAMETER, "zero amounts in \"to\" param");
+    }
+
+    if (request.params[0].get_obj().empty()) { // autofinding
+        CAccounts foundWalletAccounts = FindAccountsFromWallet(pwallet);
+
+        std::string selectionParam = request.params[2].isStr() ? request.params[2].get_str() : "pie";
+        AccountSelectionMode selectionMode = ParseAccountSelectionParam(selectionParam);
+
+        msg.from = SelectAccountsByTargetBalances(foundWalletAccounts, sumTransfersTo, selectionMode);
+
+        if (msg.from.empty()) {
+            throw JSONRPCError(RPC_INVALID_REQUEST,
+                                   "Not enough balance on wallet accounts, call utxostoaccount to increase it.\n");
+        }
+    }
+    else {
+        msg.from = DecodeRecipients(pwallet->chain(), request.params[0].get_obj());
+    }
+
+    // encode
+    CDataStream markedMetadata(DfTxMarker, SER_NETWORK, PROTOCOL_VERSION);
+    markedMetadata << static_cast<unsigned char>(CustomTxType::AnyAccountsToAccounts)
+                   << msg;
+    CScript scriptMeta;
+    scriptMeta << OP_RETURN << ToByteVector(markedMetadata);
+
+    if (scriptMeta.size() > nMaxDatacarrierBytes) {
+        throw JSONRPCError(RPC_VERIFY_REJECTED, "The output custom script size has exceeded the maximum OP_RETURN script size."
+                                                "It could have happened because too many \"from\" or \"to\" accounts balances."
```

Comment:
"It could have happened" change to "It may"

Reply:
Done